examples/custom_diffusion/README.md

```diff
@@ -277,4 +277,4 @@ To save even more memory, pass the `--set_grads_to_none` argument to the script.
 More info: https://pytorch.org/docs/stable/generated/torch.optim.Optimizer.zero_grad.html
 
 ## Experimental results
-You can refer to [our webpage](https://www.cs.cmu.edu/~custom-diffusion/) that discusses our experiments in detail. 
\ No newline at end of file
+You can refer to [our webpage](https://www.cs.cmu.edu/~custom-diffusion/) that discusses our experiments in detail. We also released a more extensive dataset of 101 concepts for evaluating model customization methods. For more details please refer to our [dataset webpage](https://www.cs.cmu.edu/~custom-diffusion/dataset.html).
```

Comment:
Would be also cool to open an organization on the Hugging Face Hub and upload the datasets, models, etc. there. 

An example Hub organization looks like this: https://hf.co/google. 

Happy to provide guidance regarding setting one of these up on our Slack channel.

Cc: @patrickvonplaten @osanseviero

Reply:
Sounds great. I will look into it this week and let you know if need any help. 
Thanks for the pointer.